### DISCUSSION TITLE
How to execute "pkexec dboublecmd" with  doublecmd.desktop in linux?

### DISCUSSION THREAD

**Discussion by jack-liew** [doublecmd/doublecmd#915] [General]
I can executed "pkexec env DISPLAY=$DISPLAY XAUTHORITY=$XAUTHORIT doublecmd" in bash, but it cannot work  in .desktop shortcut .

What should i do?
--
Thanks

**Comment by alexx2000** [comment]
#920

**Comment by Skif-off** [comment]
@jack-liew 
I apologize for my curiosity :) Why are you doing it? (In most cases, DC may request elevation.)

**Reply by jack-liew** [reply]
> @jack-liew I apologize for my curiosity :) Why are you doing it? (In most cases, DC may request elevation.)

Because when I use doublecmd, I definitely need root permission. But it's very annoying every time I enter a password,
